src/borg/item.pyx

```diff
@@ -736,6 +739,16 @@ class ItemDiff:
             mode2 = stat.filemode(self._item2.mode)
             return ({"type": "mode", "old_mode": mode1, "new_mode": mode2}, '[{} -> {}]'.format(mode1, mode2))
 
+    def _time_diffs(self):
+        changes = []
+        attrs = ["ctime", "mtime"]
```

Comment:
Sometimes there is also "birthtime", but not sure if we should check it.

Guess we better not check "atime", it changes often and is not that interesting.